katas/content/superdense_coding/alice_sends_message/solution.md

```diff
@@ -0,0 +1,33 @@
+Recall that we learnt how to prepare all Bell states in "Preparing Quantum States" kata. This is slightly advanced version of that task demonstrating the "power of entanglement", where operations applied to only one qubit affect the state of the system in interesting way.
+
+Let's use the below Bell state we prepared in the previous task:
+
+$$\frac{1}{\sqrt{2}} \big(\ket{00} + \ket{11} \big)$$
+
+We can transform it according to the value of bits:
+
+- `(0, 0)`: $\ket{\Phi^{+}} = \frac{1}{\sqrt{2}} (\ket{00} + \ket{11})$
```

Comment:
These two lists are rendered as one 8-point list, which looks a bit odd. I would merge the pairs of bullet points that describe the same scenario into one point each, so that each value of message is described and handled right away

Reply:
Nice catch. Updated solution.